util/src/main/java/io/kubernetes/client/Exec.java

```diff
@@ -280,7 +283,7 @@ protected void handleMessage(int stream, InputStream inStream) throws IOExceptio
             }
 
             @Override
-            public void failure(Exception ex) {
```

Comment:
Why is this changing?

Reply:
interface change, or the code doesn't compile